pkg/tests/integration/integration_test.go

```diff
@@ -17,6 +17,9 @@ func Test(t *testing.T) {
 	} else {
 		config.LoadConfig(configPath)
 	}
-	ontology.TestOntology(t)
-	bitcoin.TestBitcoin(t)
+	platform.Init()
+
+	//ontology.TestOntology(t)
+	//bitcoin.TestBitcoin(t)
```

Comment:
any reason to disable?

Reply:
sorry I did forget, but it's already enabled now